### ISSUE TITLE
Python

### ISSUE THREAD

**Issue by rikutarouprime14** [rikutarouprime14/Simple#1]
# Program kalkulator sederhana
def tambah(a, b):
    return a + b

def kurang(a, b):
    return a - b

def kali(a, b):
    return a * b

def bagi(a, b):
    if b != 0:
        return a / b
    else:
        return "Tidak bisa dibagi oleh nol"

angka1 = 10
angka2 = 5

print("Penjumlahan:", tambah(angka1, angka2))
print("Pengurangan:", kurang(angka1, angka2))
print("Perkalian:", kali(angka1, angka2))
print("Pembagian:", bagi(angka1, angka2))

**Comment by Pranshubatham76** [comment]
What problem you face in this?

**Comment by smth-27** [comment]
If there is any problem i'd be due to indentation, else the code looks fine to me!

**Comment by xiazhengtao** [comment]
The provided code is a simple Python script that defines four functions (tambah, kurang, kali, and bagi) for basic arithmetic operations (addition, subtraction, multiplication, and division). It then initializes two variables (angka1 and angka2) with the values 10 and 5, respectively. Finally, the script calls each of the functions with these two numbers and prints the results.

**Comment by atharv-rem** [comment]
yes your code is correct, the return statements belong to each function mentioned above so it should be indented(give a tab space)
```
def tambah(a, b):
      return a + b

def kurang(a, b):
      return a - b

def kali(a, b):
      return a * b

def bagi(a, b):
      if b != 0:
            return a / b
      else:
            return "Tidak bisa dibagi oleh nol"

angka1 = 10
angka2 = 5

print("Penjumlahan:", tambah(angka1, angka2))
print("Pengurangan:", kurang(angka1, angka2))
print("Perkalian:", kali(angka1, angka2))
print("Pembagian:", bagi(angka1, angka2))
```
